enumeratum-play-json/src/test/scala/enumeratum/Dummy.scala

```diff
@@ -10,3 +10,16 @@ object Dummy extends Enum[Dummy] with PlayJsonEnum[Dummy] {
   case object c extends Dummy
   val values = findValues
 }
+
+/**
+  * Created by dbuschman on 03/20/2018
+  */
+sealed trait InsensitiveDummy extends EnumEntry
```

Comment:
This name is genius. Insensitive Dummy 😆

Reply:
Yep! It was DummyInsensitive for about 10 seconds, but then ...